db/CSV_Writers/Review_Utils/reviewsCSVWriter.js

```diff
@@ -0,0 +1,31 @@
+const { colors, path, performance } = require('../packages.js');
```

Comment:
this might be a little nit-picky but the naming used for this file is a little misleading. if i follow correctly, this exported function uses 1 imported function to write a csv and another imported function to insert records into the db, correct?

Reply:
if you are talking about reviewsCSVWriter.js then yes it writes and inserts, need to update the name since it is a bit misleading, good call!